man/systemd-measure.xml

```diff
@@ -175,6 +175,19 @@
         <xi:include href="version-info.xml" xpointer="v252"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--private-key=<replaceable>PATH/URI</replaceable></option></term>
+        <term><option>--signing-engine=<replaceable>NAME</replaceable></option></term>
+        <term><option>--certificate=<replaceable>PATH</replaceable></option></term>
```

Comment:
hmm, i am pretty sure it owuld be really confusing if --certificate= wouldn't work if you do not use engines... i.e. i think this should be generalized.

Reply:
and as above, please name this --signing-provider=

and yes, we should change this in ukify eventually too